docs/user/generateIA.md

```diff
@@ -0,0 +1,59 @@
+# Generate an Implementation Artifact for a Node Type Interface
+
+To specify what a node type should do, the user can define an *interface* and the *operations* provided by this interface. 
+Once the operations of a node type are defined, artifacts (e.g., shell scripts, .war files) implementing these operations need to be modeled. 
+For this, Winery provides an option to **automatically generate a stub java maven project** to build a **.war file** for a defined interface. 
+This can be done in the *Interface* tab of the node type view by pressing *Generate Implementation Artifact*. 
+A node type implementation will be automatically generated as well, if it does not exist yet.
+Once you get a confirmation that the artifact was generated, please press *Save*.
+
+ ![3-AddUbuntuNodeType](graphics/modeling/3-AddUbuntuNodeType.jpg)
```

Comment:
Maybe also highlight the *Generate Implementation Artifact* button in the screenshot